codecov.yml

```diff
@@ -1,34 +1,39 @@
 codecov:
-  disable_default_path_fixes: yes
+  disable_default_path_fixes: true
   notify:
-    wait_for_ci: yes
-
+    wait_for_ci: true
+  require_ci_to_pass: true
+comment:
+  behavior: default
+  layout: header, diff, changes, uncovered, tree
+  show_carryforward_flags: false
 coverage:
+  precision: 1
+  range:
+  - 75.0
+  - 100.0
+  round: down
   status:
+    changes: false
+    default_rules:
+      flag_coverage_not_uploaded_behavior: include
+    patch:
+      default:
+        target: 60.0
     project:
       default:
-        enabled: yes
+        enabled: true
         target: auto
-        threshold: 1%
-    patch:
-      default:
-        target: 60%
-    changes: no
-  precision: 1
-  round: down
-  range: "75...100"
-
-comment:
-  layout: "header, diff, changes, uncovered, tree"
-  behavior: default
-
+        threshold: 1.0
 flags:
   engine:
     paths:
-      - "/home/runner/work/fhircore/fhircore/android/engine"
+    - ^/home/runner/work/fhircore/fhircore/android/engine.*
   geowidget:
     paths:
-      - "/home/runner/work/fhircore/fhircore/android/geowidget"
+    - ^/home/runner/work/fhircore/fhircore/android/geowidget.*
   quest:
     paths:
-      - "/home/runner/work/fhircore/fhircore/android/quest"
+    - ^/home/runner/work/fhircore/fhircore/android/quest.*
```

Comment:
Should these path definitions be starting with a caret?

Reply:
yea maybe not, I copied this directly from here, which was the last valid run, https://app.codecov.io/github/opensrp/fhircore/commit/5182221f688cfb24ae13720bb50221c2e040107e